test/e2e/workload_clusters_test.go

```diff
@@ -20,6 +20,17 @@ func runWorkloadClusterFlow(test *framework.MulticlusterE2ETest) {
 	test.DeleteManagementCluster()
 }
 
+func runWorkloadClusterFlowWithGitOps(test *framework.MulticlusterE2ETest, clusterOpts ...framework.ClusterE2ETestOpt) {
+	test.CreateManagementCluster()
+	test.RunInWorkloadClusters(func(w *framework.WorkloadCluster) {
```

Comment:
do you have thoughts about how we could upgrade a single workload cluster via gitops in a test which has multiple workload clusters?

Reply:
we can potentially have a method similar to https://github.com/aws/eks-anywhere/blob/main/test/framework/multicluster.go#L35
Something like
```
RunInWorkloadCluster(clusterName string, flow func(*WorkloadCluster))
```
and use it in the GitOps flow.

Just seems not a significant test case to test a single workload cluster in multiple workload clusters settings.